python/observation-publisher/publisher/metric.py

```diff
@@ -0,0 +1,47 @@
+from pandas import Timestamp
+from prometheus_client import Gauge
+
+
+class MetricWriter(object):
```

Comment:
Should we add metrics number of records that has been published to arize? So we can see the discrepancy between data from arize and ours